minor doc improvements/additions (#204)
brainglobe/brainglobe.github.io: docs/source/documentation/brainglobe-atlasapi/adding-a-new-atlas.md

```

Once an atlas is created with BrainGlobe’s atlas generation tools, it can be used with most software from the BrainGlobe software suite.

> These include [brainrender](/documentation/brainrender/index) and [brainrender-napari](tutorials/visualise-atlas-napari) which provide convenient GUIs for visually inspecting the generated atlas meshes.

Reviewer: Original brainrender does't have a GUI anymore & the link needs a `/`
```suggestion
These include [brainrender](/documentation/brainrender/index) and [brainrender-napari](/tutorials/visualise-atlas-napari) which provide a convenient way to visually inspect the generated atlas meshes.
```

For example, you can visualise various parts of the atlas next to each other by running the script below, replacing the content between `<>` in the code.

```python
from brainrender_napari.napari_atlas_representation import NapariAtlasRepresentation
from brainglobe_atlasapi import BrainGlobeAtlas
import napari

if __name__ == "__main__":
    viewer = napari.Viewer()
    viewer.dims.ndisplay = 3
    napari_atlas = NapariAtlasRepresentation(BrainGlobeAtlas("<newatlas_resolutionum>"), viewer)
    napari_atlas.add_additional_reference("<additional_reference>")
    napari_atlas.add_to_viewer()
    napari_atlas.add_structure_to_viewer("<structure_id>")
    napari.run()

> ```

Reviewer: This doesn't seem completely clear to me. An example with real args (rather than replace`<>`) might be clearer. May just be me though.

Author: Had a go at clarifying
